end_to_end_tests/openapi.json

```diff
@@ -888,6 +766,53 @@
           }
         }
       }
+    },
+    "/location/query/optionality": {
+      "description": "Test what happens with various combinations of required and nullable in query parameters.",
+      "get": {
+        "tags": [
+          "location"
+        ],
+        "parameters": [
+          {
+            "name": "not_null_required",
+            "required": true,
+            "schema": {
+              "type": "string",
```

Comment:
Can you include a model or datetime-type parameter, so we can verify the JSON serialization works as expected?